api/amendedJurisdictions/index.ts

```diff
@@ -4,26 +4,33 @@ import { config } from '../config'
 import { http } from '../lib/http'
 import { setHeaders } from '../lib/proxy'
 
+/**
+ * Manually filtering returned jurisdictions
+ * to make available jurisdiction in filters array only
+ */
 export async function getJurisdictions(req: express.Request, res: express.Response) {
+    console.log(req.url)
```

Comment:
Do we want to keep stray console.logs in the code?

Reply:
Forgot to re-change base to master after rebase. This file is not part of this PR.